File: server/src/main/java/de/zalando/zally/util/WordUtil.java
package de.zalando.zally.util;

import javatools.parsers.PlingStemmer;

import java.util.HashSet;
import java.util.Set;

import static java.util.Arrays.asList;

public class WordUtil {
    private static final Set<String> PLURAL_WHITELIST = new HashSet<>(asList("vat", "apis"));

    public static boolean isPlural(String word) {
        if (PLURAL_WHITELIST.contains(word)) {
            return true;
        }
        return PlingStemmer.isPlural(word);
    }
}

File: server/src/main/java/de/zalando/zally/util/WordUtil.kt
package de.zalando.zally.util

import javatools.parsers.PlingStemmer

object WordUtil {
    private val PLURAL_WHITELIST = hashSetOf("vat", "apis")

    fun isPlural(word: String): Boolean = PLURAL_WHITELIST.contains(word) || PlingStemmer.isPlural(word)
}
